node_modules/oae-content/lib/api.js

```diff
@@ -482,60 +483,73 @@ var _createContent = function(ctx, contentId, revisionId, resourceSubType, displ
         return callback(validator.getFirstError());
```

Comment:
Need to update the JSDocs in this function as well (`additionalMembers`, as well as indicating what is optional)